proto/path/qos/evm.proto

```diff
@@ -78,7 +91,7 @@ message EVMChainIDResponse {
   bool valid = 2 [(metadata.semantic_meaning) = "Response validity status"];
 
   // Why the response failed QoS validation
-  optional EVMResponseInvalidReason invalid_reason = 3 [(metadata.semantic_meaning) = "Validation failure reason"];
+  optional EVMResponseValidationError response_validation_error = 3 [(metadata.semantic_meaning) = "Validation failure type"];
```

Comment:
NIT: You change one of these to `type` but kept the other meta tags as `reason`

Reply:
Thanks for catching this. Fixed.